Sources/SwiftGenKit/Parsers/Colors/ColorsParser.swift

```diff
@@ -34,23 +34,33 @@ public enum Colors {
     var palettes = [Palette]()
     public var warningHandler: Parser.MessageHandler?
 
+    private static let subParsers: [ColorsFileTypeParser.Type] = [
+      CLRFileParser.self,
+      JSONFileParser.self,
+      TextFileParser.self,
+      XMLFileParser.self
+    ]
+
     public init(options: [String: Any] = [:], warningHandler: Parser.MessageHandler? = nil) {
       self.warningHandler = warningHandler
 
-      register(parser: CLRFileParser.self)
-      register(parser: JSONFileParser.self)
-      register(parser: TextFileParser.self)
-      register(parser: XMLFileParser.self)
+      for parser in Parser.subParsers {
+        register(parser: parser)
+      }
```

Comment:
I don't know why, but I'd have seen a `subParsers.forEach` there instead… matter of taste I guess 😛

Reply:
Of all the functional stuff in swift (`map`, `filter`, ...), `forEach` is one I never use. 🤷‍♂️